tracks/perl5/two-fer/mentoring.md

```diff
@@ -0,0 +1,102 @@
+# Mentoring
+
+This is a great opportunity to prepare for the track with attention to style conventions.
+This is currently the first reviewed problem on the Python track, and serves to
+make sure the student understands unit testing and Test Driven Design.
+
+## Reasonable Solutions
+
+```perl
+package TwoFer;
+use strict;
+use warnings;
+use Exporter 'import';
+our @EXPORT_OK = qw(two_fer);
+sub two_fer {
+    my $name = shift || "you";
+    return "One for $name, one for me.";
+}
+1;
+```
+
+```perl
+package TwoFer;
+use strict;
+use warnings;
+use Exporter 'import';
+our @EXPORT_OK = qw(two_fer);
+sub two_fer {
+    return "One for " . (shift || "you") . ", one for me.";
+}
+1;
+```
+
+```perl
+package TwoFer;
+use strict;
+use warnings;
+use Exporter 'import';
+our @EXPORT_OK = qw(two_fer);
+use experimental qw(signatures);
+sub two_fer ($name = 'you') {
+    return "One for $name, one for me.";
+}
+1;```
+## Common Suggestions
```

Comment:
```suggestion
## Common Suggestions

```

Blank line after heading, and a blank line around the "different" things.  It helps to denote paragraphs and should exist around code blocks, lists, etc.